Gopkg.lock

```diff
@@ -2,112 +2,87 @@
 
 
 [[projects]]
-  digest = "1:6b21090f60571b20b3ddc2c8e48547dffcf409498ed6002c2cada023725ed377"
```

Comment:
Looks like these changes are due to an old version of `dep` 
@iamemilio can you please update your `dep` and run `dep ensure` again please?